.github/workflows/docker_publish.yml

```diff
@@ -108,10 +127,17 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Store image name
+      - name: Configure docker image tag
         if: ${{ github.repository_owner == 'svalinn' }}
         run: |
-          echo "image_base_tag=ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler }}-geant4_${{ matrix.geant_version }}-hdf5_${{ matrix.hdf5_versions }}-moab_${{ matrix.moab_versions }}" >> "$GITHUB_ENV"
+          image_base_tag=ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_version }}-${{ matrix.compiler}}-hdf5_${{ matrix.hdf5_version}}-moab_${{ matrix.moab_version }}
+          if [ ${{ matrix.geant4_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-geant4_${{ matrix.geant4_version }}
+          fi
+          if [ ${{ matrix.double_down_version }} != "OFF" ] ; then
+            image_base_tag=${image_base_tag}-double_down_${{ matrix.double_down_version }}
+          fi
```

Comment:
update to conform with comments above

Reply:
You mean to add some documentations there how that works?

Reply:
I mean to use a single variable to either turn double down off or set the version.

Reply:
Once again, we don't need the logic here if we always include geant and DD in the tag name